configure/xlang/experimental/bash/xlang-bash.Deployment.yaml

```diff
@@ -3,6 +3,9 @@ kind: Deployment
 metadata:
   annotations:
     description: LSP server for bash (used for live requests).
+  labels:
+    app: xlang-bash
```

Comment:
I am not sure if this is desirable. Services care about this, but it doesn't make sense for a service to be aware of a deployment? Probably doesn't do any harm, but maybe work checking

Reply:
https://github.com/sourcegraph/deploy-sourcegraph/pull/58/commits/31c5cd762706253ae6e15ba61bb2fc06c5daf1c1